src/press/templatetags/press_url.py

```diff
@@ -15,19 +16,24 @@ def press_url(request):
 
 
 @register.simple_tag
-def svg(filename):
+def svg(filename, partial=False):
     path = filename
 
     if not path:
         return None
 
     mimetype = mimetypes.guess_type(path, strict=True)
+
     if not mimetype or mimetype[0] != 'image/svg+xml':
+        print(1)
         return mark_safe('<img src="{url}" class="top-bar-image">'.format(url=reverse('press_cover_download')))
 
     if isinstance(path, (list, tuple)):
         path = path[0]
 
+    if partial:
```

Comment:
Perhaps `relative_path` would make more sense than `partial`.
In fact, why not remove the extra argument and just:
```
if not path.startswith(settings.BASE_DIR):
    path = os.path.join(settings.BASE_DIR, path)
```
It has the same result as using passing the `partial` argument.

Reply:
Cool, latest commit should resolve this.